src/js/pie.js

```diff
@@ -21,19 +21,25 @@ https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.tx
     // 4. update the pie when the data set changes, including adding or removing slices
 
     fluid.defaults("floe.chartAuthoring.pieChart.pie", {
-        gradeNames: ["floe.chartAuthoring.totalRelaying", "floe.d3ViewComponent",  "autoInit"],
+        gradeNames: ["floe.chartAuthoring.valueBinding","floe.chartAuthoring.totalRelaying", "floe.d3ViewComponent"],
```

Comment:
Hi @waharnum - the very form of those linting messages looks puzzling and unexpected (e.g. including text like "W015" and "[L24:C5]") - and not of the kind generated by the linting task - the ones I am referring to are the ones output from the grunt plugin by running the task "grunt lint". I introduced both indentation and blatant syntax errors but got no output from the standard task - what are you running in order to generate the transcript you got above?